theme/static/js/hs-vue/geoconnex.js

```diff
@@ -828,14 +819,18 @@ const geoconnexApp = new Vue({
       geoconnexApp.setMapEvents();
       geoconnexApp.fitMapToFeatures();
     },
-    addSearchFeaturesToMap(features, collectionOverride = null) {
+    async addSearchFeaturesToMap(features, collectionOverride = null) {
       for (const feature of features) {
         // deal with collection first
         const collection = collectionOverride
           ? collectionOverride
           : feature.collection;
 
-        if(geoconnexApp.ignoredFeatures[collection.id] && geoconnexApp.ignoredFeatures[collection.id].includes(feature.id)) return;
+        if (
+          geoconnexApp.ignoredFeatures[collection.id] &&
+          geoconnexApp.ignoredFeatures[collection.id].includes(feature.id)
```

Comment:
This conditional can be simplified in one line as such: `geoconnexApp.ignoredFeatures[collection.id]?.includes(feature.id)`